docs/site/tutorials/core/index.md

```diff
@@ -0,0 +1,51 @@
+---
+lang: en
+title: 'Build large scale Node.js projects with LoopBack 4'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/core-tutorial.html
```

Comment:
@raymondfeng, since this tutorial is about building large scale Node.js project with LB4, i'm thinking whether it would be clearer if we use `large-scale` as the base of the permalink and file structure? Proposed names:
- permalink:  /doc/en/lb4/large-scale-tutorial.html vs /doc/en/lb4/core-tutorial.html
- folder: /docs/site/tutorials/large-scale/index.md vs /docs/site/tutorials/core/index.md .  (and other files too). 

WDYT?

Reply:
IMO, `large-scale` is vague here for the titles. The main idea is to illustrate how to use LB4's IoC container and extensibility stories to build large-scale Node.js applications independent of the REST API story.